src/components/Tabs/Tabs.tsx

```diff
@@ -79,9 +78,11 @@ export const Tabs = ({
     [tabs],
   );
 
+  const generatedId = useId(other.id);
+  const tabIdPrefix = String(generatedId);
   const tabIds = useMemo(
-    () => tabs.map((tab) => tab.props.id || getUID(tab)),
-    [tabs, getUID],
+    () => tabs.map((tab) => `${tabIdPrefix}-${tab.props.title}`),
+    [tabs, tabIdPrefix],
```

Comment:
Calling out diff here, because replacing our usage of `useUIDSeed` is a bit more involved. Note: you cannot call `useId()` inside a for-loop because that breaks the rules of react-hooks.